src/highlighter.js

```diff
@@ -380,12 +525,14 @@ export default function(options) {
                 while (currentNode) {
                     if (isWrappingNode(currentNode)) {
                         inlineRange = {
-                            groupId: currentNode.getAttribute(GROUP_ATTR)
+                            groupId: currentNode.getAttribute(GROUP_ATTR),
+                            c: getColorByClassName(currentNode.className)
```

Comment:
Will the name like `color` be clearer?

Reply:
Agree, but it was done to save space in DB